docs/src/app/components/prop-type-description.css

```diff
@@ -21,18 +21,18 @@
 
 .propTypeDescription table td {
   color: #266d90;
-  font-family: Menlo,Monaco,Consolas,"Courier New",monospace;
-  font-size: 90%;
+  font-family: "Roboto", sans-serif;
```

Comment:
That's fine if you want Roboto for type and deault (it's currently fixed-width).